enterprise/backend/src/metabase_enterprise/sso/integrations/saml.clj

```diff
@@ -109,14 +111,24 @@
   (api/check (sso-settings/saml-enabled)
     [400 (tru "SAML has not been enabled and/or configured")]))
 
+(defn- has-host? [uri]
+  (try
+    (-> uri URI. .getHost some?)
+    (catch MalformedURLException _ false)))
```

Comment:
Hmm. It seems `URI.` cannot throw `MalformedURLException`, it can throw `URISyntaxException`.